client/cmd/apps.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/deis/pkg/prettyprint"
+	"github.com/deis/deis/pkg/prettyprint"
```

Comment:
can you use `github.com/deis/pkg/prettyprint` instead. it pulls into far fewer dependencies and still does what you need.